### ISSUE TITLE
Not able to pull the docker image from the registry 

### ISSUE THREAD

**Issue by srujansagi** [IBM/domino-docker#44]
Hello,

I'm  getting certificate error when trying to poll docker image from this registry

`docker pull registry.domino-lab.net:5000/hclcom/domino:latest11
Error response from daemon: Get https://registry.domino-lab.net:5000/v2/: x509: certificate is valid for *.nashcom.de, nashcom.de, not registry.domino-lab.net `

Any help is greatly appreciated.

**Comment by Daniel-Nashed** [comment]
The registry is a lab registry for a workshop. there is no public registry to pull the image.  
The Domino image needs to be build in your environment from the Domino web kit.   
Usually you get the software from Flexnet. There is no free download available.  And no public image.
What you are looking into is the lab material I did for a commercial workshop, where I decided to open source the scripts etc. And the registry is our lab registry for the workshop. I left the registry as an example in the lab setup guide. 

I hope this helps?! 

We can't set anything up for the community, because HCL Domino is commercial software, that we can't re-distribute.

-- Daniel
